source/_patterns/02-molecules/additional-content/additional-content.twig

```diff
@@ -0,0 +1,30 @@
+{#   
```

Comment:
this is not a molecule.  Only locations can use this.  Put them in 04-templates.

Use "news" as reference: https://github.com/JudicialCouncilOfCalifornia/courtyard/tree/2.x/source/_patterns/04-templates